arcgis-ios-sdk-samples/Route & Directions/Route around barriers/RouteParametersViewController.swift

```diff
@@ -56,13 +49,26 @@ class RouteParametersViewController: UITableViewController {
         switch sender {
         case findBestSequenceSwitch:
             routeParameters?.findBestSequence = sender.isOn
+            if sender.isOn {
```

Comment:
This looks good. One thing I would suggest is to adjust the size of the popover to show the new sections. Here's one way of doing that:

```swift
let sections: IndexSet = [1]
tableView.performBatchUpdates({
    if sender.isOn {
        tableView?.insertSections(sections, with: .fade)
    } else {
        tableView?.deleteSections(sections, with: .fade)
    }
}, completion: { (finished) in
    guard finished else { return }
    self.preferredContentSize.height = self.tableView.contentSize.height
})
```

And it looks something like this:

![screen recording 2018-11-30 at 9 59 41 am 2018-11-30 10_00_29](https://user-images.githubusercontent.com/2257493/49306569-6fe76c80-f487-11e8-8f0a-81e4d679e583.gif)